src/Microsoft.Android.Templates/android-activity/.template.config/template.json

```diff
@@ -21,5 +21,18 @@
       "replaces": "AndroidApp1",
       "type": "parameter"
     }
-  }
+  },
+  "postActions": [
+    {
+      "id": "openInEditor",
+      "condition": "(HostIdentifier != \"dotnetcli\" && HostIdentifier != \"dotnetcli-preview\")",
+      "description": "Opens Activity1.cs in the editor",
+      "manualInstructions": [],
+      "actionId": "84C0DA21-51C8-4541-9940-6CA19AF04EE6",
+      "args": {
+        "files": "1"
```

Comment:
Did some testing of this with an existing install and the steps below. The file open action worked for me after changing this index to `0`, as we only include one file in this template

1) Unzip `C:\Program Files\dotnet\template-packs\Microsoft.Android.Templates.34.0.52.nupkg`
2) Edit `C:\Program Files\dotnet\template-packs\Microsoft.Android.Templates.34.0.52\content\android-activity\.template.config\template.json`
3) Repack with `nuget pack C:\Program Files\dotnet\template-packs\Microsoft.Android.Templates.34.0.52\Microsoft.Android.Templates.nuspec`
4) Replace `C:\Program Files\dotnet\template-packs\Microsoft.Android.Templates.34.0.52.nupkg` with the new package